src/vs/workbench/contrib/terminalContrib/suggest/browser/lspCompletionProviderAddon.ts

```diff
@@ -21,17 +22,20 @@ export class LspCompletionProviderAddon extends Disposable implements ITerminalA
 	private _provider: CompletionItemProvider;
 	private _textVirtualModel: IReference<IResolvedTextEditorModel>;
 	private _lspTerminalModelContentProvider: LspTerminalModelContentProvider;
+	private _logService: ITerminalLogService;
 
 	constructor(
 		provider: CompletionItemProvider,
 		textVirtualModel: IReference<IResolvedTextEditorModel>,
 		lspTerminalModelContentProvider: LspTerminalModelContentProvider,
+		@ITerminalLogService logService: ITerminalLogService,
```

Comment:
Would this be the proper log service I should be using?

Reply:
but yes, this is the right log service

Reply:
@anthonykim1 that's a wrapper around `ILogService` that puts everything in the `Terminal` output channel